README.md

```diff
@@ -4,364 +4,414 @@ The purpose of the ODM is to support wastewater-based surveillance and epidemiol
 
 The ODM supports data sharing in two ways:
 
-1. **Data sharing schema** - The ODM will have a schema that describes what data can be shared with one or more partners or users.
-2. **Data filter based on the data sharing schema** - The ODM will support an open source method for filtering data tables using the data sharing schema.
+1.  **Data sharing schema** - The ODM will have a schema that describes what data can be shared with one or more partners or users.
+2.  **Data filter based on the data sharing schema** - The ODM will support an open source method for filtering data tables using the data sharing schema.
 
-The data sharing schema will be a csv file (sharing.csv) where each row in the file corresponds to one variable in a ODM table. Attributes in the row describe who the data is shared with and what data is excluded. See below for an example.
+The data sharing schema will be a csv file (sharing.csv) where each row in the `sharing` file corresponds to a header or table in the PHES-ODM. Attributes in the row describe who the data is shared with and what data is included. See below for an example.
 
-The data filter is a Python module (or function) that excludes data based on a data sharing schema. The function accepts an ODM data tables and sharing schema. The function excludes (filters) data variables and rows according to the schema rules. The function then returns a data table with only the data that is to be shared. This new, returned data is ready to be shared and used with a partner.
+| ruleId | sharedWith | table    | variable   | direction | ruleValue               | notes       |
```

Comment:
OK, interesting. I had envisioned a CSV for each organization/entity that would have shared data. Meaning, a database could have multiple sharing files, representing sharing rules for multiple organizations. Of course, that would still be possible, and I expect most data custodians would want separate sharing files.

Reply:
I could write this in that if you have similar rules across orgs you can use a single file, or you can split it up across multiple sharing.csv files. My thinking was that the sharing schema would be used by the library to generate multiple outputs, so it wouldn't be necessary to have multiple sheets. But I'm not sure what the user preference would be.

Reply:
Opting to allow single or multiple, at the user's discretion.